### PR TITLE
fix(video): fix ref to `url` in resolver

### PR DESCRIPTION
**Pull request by damassi**
Fixes an issue with a field rename in the Video resolver.

### CODE REVIEW

File: src/schema/v2/types/Video.ts [hunk 1]
```diff
@@ -8,28 +8,37 @@ import {
...
-    id: {
-      type: new GraphQLNonNull(GraphQLID),
-      resolve: ({ url }) => {
-        return uuid(url, uuid.URL)
+interface VideoTypeProps {
+  id: string
+  playerUrl: string
+  height: number
+  width: number
+}
```
Comment (dzucconi) [inline src/schema/v2/types/Video.ts:16]: It'd be good to annotate the return of the function that constructs this object with this type. (Or import the `ReturnType` of that function here instead.)

Comment (damassi) [reply src/schema/v2/types/Video.ts:16]: Not sure that I follow? Like, the return type of `uuid`? (Can follow-up)

Comment (dzucconi) [reply src/schema/v2/types/Video.ts:16]: `getFigures` returns:

Comment (dzucconi) [reply src/schema/v2/types/Video.ts:16]: so make that like: `NormalizedImageData | VideoData` and then import `VideoData` here.

Comment (damassi) [reply src/schema/v2/types/Video.ts:16]: Oh I see! Will do 👍

**Pull request merged**